virttest/qemu_vm.py

```diff
@@ -1890,6 +1890,14 @@ def sort_key(dev):
                                       dynamic=True)
                 devices.insert(pvpanic_dev)
 
+        # Add vmcoreinfo device
+        if params.get("vmcoreinfo") == "yes":
+            vmcoreinfo_params = {"backend": "vmcoreinfo"}
+            vmcoreinfo_dev = qdevices.QCustomDevice("device",
+                                                    params=vmcoreinfo_params,
+                                                    backend="backend")
```

Comment:
```suggestion
            vmcoreinfo_dev = qdevices.QDevice("vmcoreinfo")
```

Reply:
Thanks so much,  I'll update it~

Reply:
@vivianQizhu Updated, it's magic, test with new code, it's works~
Thanks a lot~